exchangelib/properties.py

```diff
@@ -136,8 +136,7 @@ def to_xml(self, version):
 
         # Add elements and values
         for f in self.supported_fields(version=version):
-            if f.is_read_only:
-                continue
+            # Nylas likes to save data in read_only fields
```

Comment:
What do you mean by this? What sort of data is going to be stored in the read_only fields? Also, I wonder if after this merge is approved and the old code has gone out of site, if this comment will make sense and be necessary?

Reply:
so if you look at CalendarItem, read_only fields are things like `is_cancelled` and `my_response_type` and `organizer` --  https://github.com/nylas/exchangelib/blob/master/exchangelib/items.py#L515  And I would like the comment there because it will make someone's life easier down the line if/when we try to rebase off master and there's no note here about why this change was made, and by whom

Reply:
Thats a great point. What if the comment said "Exchangelib previously swallowed read_only fields but refactored this to keep the read_only fields so we could see things like `is_cancelled` and `organizer` for CalendarItems"